main.py

```diff
@@ -303,13 +309,178 @@ def extract_data(directory: str, has_dedicated_gpu: bool, gpu_in_cpu: bool, gui:
 
     return result
 
+def get_gpu(args):
+
+    if args.files is not None:
+        args.cpu = False
+        args.gpu = False
+        args.motherboard = False
+
+        try:
+            with open(os.path.join(os.getcwd(), args.files, "gpu_location.txt")) as f:
+                location = f.readline().lower().rstrip()
+                if location == 'mobo':
+                    args.motherboard = True
+                elif location == 'gpu':
+                    args.gpu = True
+                elif location == 'cpu':
+                    args.cpu = True
+        except FileNotFoundError:
+            pass
+
+    while not any((args.gpu, args.cpu, args.motherboard)):
+        print("\nWhere is GPU in your PC? c/g/b\n",
+              "c for integrated in CPU\n",
+              "g for discrete graphics card\n",
+              "b for integrated in the motherboard\n")
+        gpu_flag = input("Insert your choice: ")
+        if gpu_flag == 'c':
+            args.cpu = True
+        elif gpu_flag == 'g':
+            args.gpu = True
+        elif gpu_flag == 'b':
+            args.motherboard = True
+    return args.cpu, args.gpu, args.motherboard
+
+
+def print_output(output, path):
+    print("\nThe following output can be copy-pasted into the 'Bulk Add' page of the TARALLO, from '[' to ']':\n")
+    print(output)
+
+    try:
+        with open(os.path.join(path, "copy_this_to_tarallo.json"), "x") as f:
+            f.write(output)
+    except FileExistsError:
+        with open(os.path.join(path, "copy_this_to_tarallo.json"), "w") as f:
+            f.write(output)
```

Comment:
Triggering an error and then ignoring it is redundant, check [the docs](https://docs.python.org/3/library/functions.html#open).
```suggestion
    with open(os.path.join(path, "copy_this_to_tarallo.json"), "w") as f:
        f.write(output)
```